semantic_conventions/resource/redis.yaml

```
@@ -0,0 +1,11 @@
+groups:
+  - id: redis
+    prefix: redis
+    brief: >
+        A redis instance
+    attributes:
+      - id: instance
+        type: string
+        brief: >
+          Reported name of the redis instance
+        examples: ['localhost:6379']
```

```suggestion
        examples: ['localhost:6379', 'product_info_redis']
```
To make clear it cannot only be in that form but any name.

---

Done